### ISSUE TITLE
Date Changed When Restoring

### ISSUE THREAD

**Issue by TheKidRock** [kopia/kopia#3932]
Hi,
I came across Kopia and did a test backup/restore. Everything worked smoothly. However, when restoring a file, the file's actual Created and Modified dates were both changed to the dates of restore. 

Is there any way to restore the file with actual dates? Thanks for any suggestions.

**Comment by mstenzel70** [comment]
Yes there is. However, since you're not giving any information about your restore process, I cannot help you with any information.
